src/examples/forms/tiles/TilesDefault.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Tiles } from '@zendeskgarden/react-forms';
+import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+import { ReactComponent as ImageIcon } from '@zendeskgarden/svg-icons/src/16/file-image-stroke.svg';
+import { ReactComponent as PresentationIcon } from '@zendeskgarden/svg-icons/src/16/file-presentation-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin: ${p => p.theme.space.xs} 0;
```

Comment:
Prefer "push-down" `margin-top` media query styling here for all `Col` except the first. See https://garden.zendesk.com/components/split-button#size for details.